src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -199,14 +201,40 @@ let private distributeValues (valuesByAID: seq<AidHash * array<Value>>) : seq<Ai
 
   result :> seq<AidHash * Value>
 
+let private valueComparer = Value.comparer Ascending NullsFirst
+
+let private comparePerAidContributions
+  ((aidA, valuesA): AidHash * Value array)
+  ((aidB, valuesB): AidHash * Value array)
+  =
+  if valuesA.Length <> valuesB.Length then
+    valuesA.Length - valuesB.Length
+  else
+    let result =
```

Comment:
the new sorter is different by that it sorts using 3 criteria:
1. array length
2. (new!) first differing array element
3. aid

This makes the sorting more cross-platform, as we don't get sorted differently by different hashing algorithms. If the hashing algorithms are made equal, we can get rid of this change.

Reply:
NOTE: consistent hashing allowed us to get rid of this change, hence comment is `Outdated`.